static/src/javascripts/projects/common/modules/identity/consents.js

```diff
@@ -137,14 +139,40 @@ const checkAllOptOuts = (): Promise<void> =>
 const uncheckAllOptIns = (): Promise<void> =>
     toggleInputsWithSelector(optInClassName, false);
 
+const showUnsubscribeConfirmation = (): Promise<void> => {
+    const fetchElements = (): Promise<(HTMLButtonElement | HTMLDivElement)[]> =>
+        fastdom.read(() =>
+            [
+                `.${unsubscribeButtonClassName}`,
+                `.${unsubscribeSuccessMessageClassName}`,
+            ].map(_ => document.querySelector(_))
+        );
+
+    const updateVisibility = (
+        elems: (HTMLButtonElement | HTMLDivElement)[]
```

Comment:
we can destructure this to `const updateVisibility = ([button: HTMLButtonElement, message: HTMLDivElement)]): Promise<void> =>`